### ISSUE TITLE
can WebWarp and Webalign have a single shared filetype

### ISSUE THREAD

**Issue by PolarBean** [Neural-Systems-at-UIO/WebWarp#22]
It would be great if people could easily switch between webalign and webwarp and the workbench enables this, but the presence of wwrp filetype adds friction. Is it possible that both could use the waln extension and have a shared file?

**Comment by Majpuc** [comment]
Good point but we don't have time to prioritize this now as all content types have been created and reported. A change now would imply a lot of extra work.

**Comment by PolarBean** [comment]
we already have the waln filetype registered with WebWarp. WebWarp already uses waln as an input, this change is just to the extension placed on the file, and would improve user experience. Also the user would not see the waln or wwrp extension since the workbench hides this.

**Comment by PolarBean** [comment]
sweet thats good news, ill demo what i mean in the meeting tomorrow but I think it might be good to allow webwarp to save waln  files.  perhaps the default behaviour of the save button should be to save to the waln file that is currently open

**Comment by Tevemadar** [comment]
Hmm, ooops, I think I misread the original question.

**Comment by Tevemadar** [comment]
I think these requests (this one and the one on slack) should rather be revised as workbench-specific support requests. They only make sense in the context of the workbench, so we should call them like that. For example you wouldn't want to name all your images as `.png` on your hard drive just because software can recognize what's inside in other ways, but it makes sense inside the workbench, as users won't deal with the actual files there unless they really want to.
So yes, it's viable to add workbench-specific behaviors.

**Comment by PolarBean** [comment]
Amended :)

**Comment by Tevemadar** [comment]
WA/WW/LZ: supplying a `filename` parameter disables renaming operations:
* <kbd>Save As..</kbd> button is not displayed at all
* <kbd>Save</kbd> operation does not check filetype.
